notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb

```diff
@@ -54,12 +54,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    sys.path.append("../utils")
</pre><p>Should we download <code>notebook_utils</code> file instead of using it from <code>utils</code> directory? Or this is not the case and this notebook is not supposed to be standalone?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Done</p>